### ISSUE TITLE
Back/forward/reload buttons not working while viewing tabs in compact mode

### ISSUE THREAD

**Issue by Scepty-SSB** [zen-browser/desktop#1143]
### What happened?

I primarily use compact mode, and I've recently noticed that when I go to use the back/forward/reload page buttons, they won't work if the tab bar is showing. In the attatched video, I try to click the buttons several times and nothing happens.

Edit: Confirmed to still be an issue in 1.0.0-a.39

https://github.com/user-attachments/assets/0c31aa41-e258-4123-9439-ad3416ef6d4f

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0-a.34

### Severity impact

Low

### What platform are you seeing the problem on?

macOS - aarch64

**Comment by mally8** [comment]
That happens because of the padding the Sidebar uses to align it self.
I made a quick and dirty fix if anyone wanna try until the fix is implemented from Zen.

Follow the instructions here [Zen Live Editing Docs](https://docs.zen-browser.app/guides/live-editing)
and then add this to the userChrome.css

I should note. If you are using the Smaller Compact Sidebar, it will move the Sidebar a little down.
```css
@media (-moz-bool-pref: "zen.view.compact") {
  /* the 7px is pretty much a trial and error until it looked nice! */
  #navigator-toolbox {
    --calculation: calc(
      var(--zen-compact-toolbox-margin-single) + 0.15rem +
        var(--zen-element-separation) +
        (7px + (var(--toolbarbutton-inner-padding) * 2))
    );
    --padding-fix: var(--calculation) 0 0 0;
    margin: var(--padding-fix) !important;
    padding: var(--zen-compact-toolbox-margin-single) !important;
    height: calc(100% - var(--calculation)) !important;
  }
}
```

**Comment by nazmiyigitefe** [comment]
I experienced it on my Windows device as well. Thanks for the fix.

https://github.com/user-attachments/assets/b5b48889-79b8-447e-b86c-5bdd3d428710

**Comment by Anthonyy232** [comment]
+1 for this, really inconvenient since hovering near the back arrow actually opens the side bar too

**Comment by BionicBlade0** [comment]
+1 for this.

**Comment by mally8** [comment]
This seems to be fixed in update 1.0.1-a, I personally don't see any issues clicking back/forward anymore!

**Comment by Scepty-SSB** [comment]
> This seems to be fixed in update 1.0.1-a, I personally don't see any issues clicking back/forward anymore!

Agreed.
